core/tabs/system-setup/fedora/rpm-fusion-setup.sh

```diff
@@ -9,15 +9,28 @@ installRPMFusion() {
         dnf)
             if [ ! -e /etc/yum.repos.d/rpmfusion-free.repo ] || [ ! -e /etc/yum.repos.d/rpmfusion-nonfree.repo ]; then
                 printf "%b\n" "${YELLOW}Installing RPM Fusion...${RC}"
-                "$ESCALATION_TOOL" "$PACKAGER" install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm
+                "$ESCALATION_TOOL" "$PACKAGER" install "https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm" "https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm"
                 "$ESCALATION_TOOL" "$PACKAGER" config-manager --enable fedora-cisco-openh264
                 printf "%b\n" "${GREEN}RPM Fusion installed${RC}"
             else
                 printf "%b\n" "${GREEN}RPM Fusion already installed${RC}"
             fi
-            ;;
-        *)
-            printf "%b\n" "${RED}Unsupported distribution: $DTYPE${RC}"
+
+            printf "%b\n" "${YELLOW}Do you want to enable RPM Fusion? (Y/N): ${RC}"
+            read -r choice
+            case "$choice" in
+                y | Y)
+                    "$ESCALATION_TOOL" dnf config-manager --set-enabled rpmfusion-nonfree-updates
+                    "$ESCALATION_TOOL" dnf config-manager --set-enabled rpmfusion-free-updates
+                    printf "%b\n" "${GREEN}RPM Fusion enabled${RC}"
+                    ;;
+                n | N)
+                    printf "%b\n" "${RED}Skipping RPM Fusion enablement${RC}"
+                    ;;
+                *)
+                    printf "%b\n" "${RED}Invalid Option! RPM Fusion not enabled${RC}"
+                    ;;
+            esac
```

Comment:
Doesn't enable for invalid option anyway.

```suggestion
            printf "%b\n" "${YELLOW}Do you want to enable RPM Fusion? (y/N): ${RC}"
            read -r choice
            case "$choice" in
                y|Y)
                    "$ESCALATION_TOOL" dnf config-manager --set-enabled rpmfusion-nonfree-updates
                    "$ESCALATION_TOOL" dnf config-manager --set-enabled rpmfusion-free-updates
                    printf "%b\n" "${GREEN}RPM Fusion enabled${RC}"
                    ;;
                *)
                    printf "%b\n" "${RED}Skipping RPM Fusion enablement${RC}"
                    ;;
            esac
```